script/dataTypes.js

```diff
@@ -0,0 +1,74 @@
+// mimimum
+
+//Виконай додавання 0,1 і 0,2 добийся математично правильної відповіді.
+let sum = 0.1 + 0.2;
+console.log(+sum.toFixed(1));
+
+// Виконай додавання рядка "1" і цифри 2 (обидві операнди повинні бути в змінних), добийся математично правильної відповіді.
+let a = "1";
+let b = 2;
+console.log(+a + b);
+
+//Користувач вказує обсяг флешки в Гб. Програма повинна порахувати скільки файлів розміром в 820 Мб поміщається на флешку.
+let volumeFlashGB = prompt("Enter the volume of the flash drive in gigabytes (GB)");
+const FILE_SIZE = 820;
+let numberOfFiles = Math.floor(volumeFlashGB * 1000 / FILE_SIZE);
+console.log(`The ${volumeFlashGB} GB flesh drive will hold ${numberOfFiles} file(s) with a size of ${FILE_SIZE} MB.`);
+
+// standart
+
+/*Користувач вводить суму грошей в гаманці і ціну однієї шоколадки. 
+Програма виводить скільки шоколадок може купити користувач і скільки здачі у нього залишиться.*/
+
+let moneyInWallet = prompt("Enter the emount of money in the your wallet (in UAH).");
+let priceOfChocolate = prompt("Enter the price of chocolate (in UAH).");
+if (+moneyInWallet > +priceOfChocolate) {
+    let numberOfChocolates = Math.floor(moneyInWallet / priceOfChocolate);
+    let rest = moneyInWallet % priceOfChocolate;
+    console.log(`For ${moneyInWallet} UAH you can buy ${numberOfChocolates} chocolates and ${rest} UAH the rest will remain.`);
```

Comment:
We can have a scenario where `rest` can be f.e. `4.88888889`. So would be a good catch to have `toFixed(2)` or something

Reply:
ok